src/components/Config/Config.tsx

```diff
@@ -70,15 +67,27 @@ function Config() {
   };
 
   const handleConnect = useCallback(async () => {
-    checkMetamaskInstalled();
+    metamaskState.connect();
```

Comment:
what I meant is `useMetamask` already does this check so you don't need to do anything. just remove this line